BookReaderDemo/demo-internetarchive.html

```diff
@@ -8,6 +8,7 @@
     <meta name="apple-mobile-web-app-capable" content="yes">
 
     <!-- JS dependencies -->
+    <script src="https://unpkg.com/@webcomponents/custom-elements"></script>
```

Comment:
@iisa @jbuckner Do we have a web components polyfill on IA? This was needed to get things working on iOS9.

Reply:
Yeah we are using the `@webcomponents/webcomponentsjs` polyfill: https://github.com/webcomponents/polyfills/tree/master/packages/webcomponentsjs

Reply:
Sweet! I'll switch to that and test

Reply:
It didn't was because of this #477 , but the workaround was pretty simple.